packages/components/navigation/MiniModeNavigator.tsx

```diff
@@ -94,9 +87,9 @@ const MainTab = () => {
   );
 };
 
-export const MiniModeNavigator: React.FC = () => {
+export const MiniModeNavigator = () => {
```

Comment:
```suggestion
export const getMiniModeScreens = () => {
```

Reply:
the other name is confusing as the capitalized names are used for components and this is not a component but rather a function that returns jsx, also it's not a navigator anymore